src/lib/utils/audio.ts

```diff
@@ -18,6 +18,7 @@ export const getRandomThirtySecondWindow = (duration: number) => {
 };
 
 export interface SpotifyAdditions {
+	artists?: string[];
```

Comment:
Don't make this optional and remove the non null assertions in other places

Reply:
When you make it non-optional I get the following problem in Queue.ts at the line: `[nextTrackFull] = response.tracks;`
```
Type 'TrackInfo' is not assignable to type 'TrackInfo & SpotifyAdditions'.
        Property 'artists' is missing in type 'TrackInfo' but required in type 'SpotifyAdditions'
```

Reply:
That's a valid problem. You need to standardized `.author` and `.artists`. maybe make it all use `.artists` and for lavalink do `[info.author]`

Reply:
Sorry I'm not sure what you mean by "for lavalink do [info.author]"

Reply:
Change:
```ts
type ExtendedTrack = Track & { info: TrackInfo & SpotifyAdditions };
```
To
```ts
type ExtendedTrack = Track & { info: Omit<TrackInfo, 'author'> & SpotifyAdditions };
```

And then on line 87, instead of directly using the object returned from `decode`, modify it to use the `artists` property. Something like:
```ts
			this.nowPlaying = nextTrackFull ?? { track: nextTrack as string, info: await this.player.node.decode(nextTrack as string) };
```
To:
```ts
const trackInfo = !nextTrackFull && await this.player.node.decode(nextTrack as string);
this.nowPlaying = nextTrackFull ?? { track: nextTrack as string, info: { ...trackInfo, artists: [trackInfo.author]  } };
```